pwndbg/prompt.py

```diff
@@ -5,17 +5,38 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import os
+import subprocess
+
 import gdb
 
 import pwndbg.events
 import pwndbg.memoize
 import pwndbg.stdio
 
-msg = "Loaded %i commands.  Type pwndbg [filter] for a list." % len(pwndbg.commands._Command.commands)
-print(pwndbg.color.red(msg))
 
+def show_version():
```

Comment:
Yes. Should we launch this command on startup then?

What about situation when git is not there? Should I detect it and output "Can't fetch git commit id to display version" or try to fetch version first and don't load the command at all if git is not there?